docs/global/modules/core/pages/reference/component-bulk-model-sync-gradle.adoc

```diff
@@ -0,0 +1,145 @@
+== Gradle Plugin for synchronizing models between model-api endpoints
+:navtitle: `bulk-model-sync-gradle`
+
+:tip-caption: 🔗 Quick Links
+[TIP]
+--
+https://github.com/modelix/modelix.core[Repository^] | https://github.com/modelix/modelix.core/blob/main/model-sync-gradle/build.gradle.kts[buildfile^]
+--
+
+== Overview
+
+The `bulk-model-sync` Gradle plugin synchronizes models between `model-api` endpoints.
+A common example would be synchronizing a local MPS project to a `model-server`.
+
+IMPORTANT: For correct synchronization the used model-server is required to use IDs for roles. This can be achieved by starting the model-server with the `-useroleids` flag.
```

Comment:
```suggestion
IMPORTANT: For correct synchronization the used model-server is required to use IDs for roles.
This can be achieved by starting the model-server with the `-useroleids` flag.
```

That way the sentence per line style continues.